api/MouseWheelEvent.json

```diff
@@ -0,0 +1,208 @@
+{
+  "api": {
+    "MouseWheelEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/MouseWheelEvent",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": true
+          },
+          "chrome_android": {
+            "version_added": null
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": false
+          },
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": "6"
+          },
+          "opera": {
+            "version_added": true
+          },
+          "opera_android": {
+            "version_added": null
+          },
+          "safari": {
+            "version_added": true
+          },
+          "safari_ios": {
+            "version_added": null
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": false,
+          "deprecated": true
+        }
+      },
+      "wheelDelta": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MouseWheelEvent/wheelDelta",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": "9"
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": false,
+            "deprecated": true
+          }
+        }
+      },
+      "wheelDeltaX": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MouseWheelEvent/wheelDeltaX",
```

Comment:
Are there guidelines on invalid MDN URLs? I doubt a page would ever be created for this.

Reply:
I always put them and they automatically get rendered in red; [see here for an example](https://developer.mozilla.org/docs/Web/API/TransitionEvent#Browser_compatibility). 
The way I see it, there are 2 advantages: 
1. The reader can quickly see that there is no page (otherwise it's ambiguous).
2. When the page is added you don't need to make a pull request to add the url.